pages/common/qutebrowser.md

```diff
@@ -0,0 +1,24 @@
+# qutebrowser
+
+> A keyboard-driven, vim-like browser based on PyQt5.
```

Comment:
I can't tell if I like this or not.  At first glance, the sentences seems a bit jumbled without the comma.  Two more seconds of looking at it though and it makes sense.  🤔🤔 
```suggestion
> A keyboard-driven, vim-like browser, based on PyQt5.
```

Reply:
It's the literal description given by the project, so I just pasted it over. But like you said, read it twice it works better than with the second comma.